RiotSharpTest/RiotApiTest.cs

```diff
@@ -738,5 +738,124 @@ public void GetFeaturedGamesAsync_Test()
 
             Assert.IsNotNull(games.Result);
         }
+
+        [TestMethod]
+        [TestCategory("RiotApi")]
+        public void GetChampionMastery_Test()
+        {
+            const long lucianId = 236;
+            var championMastery = api.GetChampionMastery(Platform.NA1, id, lucianId);
+            
+            Assert.IsNotNull(championMastery);
+            Assert.IsNotNull(championMastery.PlayerId);
+            Assert.IsNotNull(championMastery.ChampionId);
+            Assert.IsNotNull(championMastery.ChampionLevel);
+            Assert.IsNotNull(championMastery.ChampionPoints);
+            Assert.IsNotNull(championMastery.LastPlayTime);
+            Assert.IsNotNull(championMastery.ChampionPointsSinceLastLevel);
+            Assert.IsNotNull(championMastery.ChampionPointsUntilNextLevel);
+            Assert.IsNotNull(championMastery.ChestGranted);
```

Comment:
If I am not mistaken all this fields should be present in response in case this champion has been played, so I thought it would be good to check them too.